ext/standard/image.c

```diff
@@ -1197,6 +1197,10 @@ int php_is_image_avif(php_stream * stream) {
 	uint32_t header_size_reversed, header_size, i;
 	char box_type[4], brand[4];
 
+	if (stream == NULL) {
+		return 0;
+	}
```

Comment:
Maybe just assert that (i.e. caller has to actually supply a stream):
````
	ZEND_ASSERT(stream != NULL);
````

Reply:
I learned something new today: there's a handy `ZEND_ASSERT`!